docs/topics/gradle.md

```diff
@@ -316,10 +293,51 @@ kotlin{
 
 </tabs>
 
-> You can use shorthand for a dependency on a Kotlin module, for example, kotlin("test") for "org.jetbrains.kotlin:kotlin-test".
+> You can use shorthand for a dependency on a Kotlin module, for example, kotlin('test') for "org.jetbrains.kotlin:kotlin-test".
 >
 {type="note"}
 
+Additionally, you can use the `kotlin-test` dependency in any shared or platform-specific source set.
+
+For Kotlin/JVM Gradle uses JUnit 4 by default; therefore, the `kotlin('test')` dependency resolves to JUnit 4.
+
+You can choose JUnit 5 or TestNG by calling 
+[`useJUnitPlatform()`]( https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useJUnitPlatform) 
+or [`useTestNG()`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useTestNG).
+The following example is for a JVM project:
```

Comment:
Perhaps we should add an MPP example since we started this section from MPP.
```
kotlin {
    jvm {
        testRuns["test"].executionTask.configure {
            useJUnitPlatform()
        }
    }
}
```

Reply:
As I remember, on the call we've discussed that there are many server-side users to whom we target the code too. You told it will be cool if there is a ready piece of code that includes as adding the dependency, as calling usage of some platform. Did I understand you incorrectly or did you change your mind? Or maybe you offer to add the additional example next to the existing one?

Reply:
Yes, you're right.
I suggest adding **one more example** because calling these tasks in MPP is not so trivial as in JVM. The copy-pastable piece of config can save a lot of time for our readers :)

Reply:
Added the example.